clients/cobol-lsp-vscode-extension/src/constants.ts

```diff
@@ -43,6 +43,7 @@ export const COPYBOOK_EXT_ARRAY = [COPYBOOK_CPY_EXT];
 export const EXTENSION_ID = "broadcommfd.cobol-language-support";
 export const TELEMETRY_DEFAULT_CONTENT = "INVALID_INSTRUMENTATION_KEY";
 export const ZOWE_EXT_MISSING_MSG = "Zowe Explorer version 1.15.0 or higher is required to download copybooks from the mainframe."
+export const INSTALL_ZOWE = "Install ZOWE Explorer";
```

Comment:
@zeibura Is this message ok? We show this when the user doesn't have ZE installed. With a button to navigate to ZE plugin.

Reply:
Zowe shouldn't be in caps, otherwise it's fine, as long as it fits on the button. If not, you can use "Install ZE".

Reply:
Updated.
![image](https://user-images.githubusercontent.com/69206564/151001014-384f34f3-8e70-464f-b91b-39ca1c338d32.png)